### PR TITLE
  Chore #37 setup environment variables

### PR DESCRIPTION
**Pull request by nili-alu**
**What does this PR do?**
configure dotenv
Config cross-env to return the environment your running in

**Description of Task to be completed**
I configured the app to .env file for hiding the sensitive information, then use of cross-env to switch environment

**How should this be manually tested?**
git clone https://github.com/atlp-rwanda/ecommerce-app-predators-bn.git

if you have already the project cloned do: 
git checkout ch-dotenv-setup-37

run npm i
npm run dev 
npm test 

**What are the relevant pivotal tracker stories?**
[card#37](https://trello.com/c/VKtMTQKx/37-setup-environments)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,8 @@
+NODE_ENV ={
+PRO_DATABASE="postgress_url"
+DEV_DATABASE="postgress_url"
```
Comment (ALVINdimpos) [inline .env]: great job

File: server.js
Comment (yobuir) [inline server.js:1]: I think this variable `NODE_ENV` is not defined in .ENV file

**Review by bryan936M — APPROVED** [review]
This is excelent work.
The completion of your task will allow us to make use of environment variables & all of their functionalities.
Your coding skills have been & will keep improving.

**Review by yobuir — APPROVED** [review]
great work

File: .env [hunk 2]
```diff
@@ -0,0 +1,8 @@
+NODE_ENV ={
```
Comment (chanceDira) [inline .env]: why there is this "{" ?

File: server.js
Comment (chanceDira) [reply server.js:1]: Yeah, exactly. @nili-alu @Vanessa-bitega you can find a value to put their

**Review by chanceDira — CHANGES_REQUESTED** [review]
Good job @nili-alu @Vanessa-bitega , just some changes. Also improve your PR description

File: .env (hunk 2 above)
Comment (nili-alu) [reply .env]: I used that to get all elements inside , this is how it looks 
NODE_ENV ={
        PRO_DATABASE="postgress_url"
        DEV_DATABASE="postgress_url"
        TEST_DATABASE="postgress_url"
}

File: server.js
Comment (nili-alu) [reply server.js:1]: this is how it looks, NODE_ENV  is set of all links we will use
NODE_ENV ={
        PRO_DATABASE="postgress_url"
        DEV_DATABASE="postgress_url"
        TEST_DATABASE="postgress_url"
}

**Review by ALVINdimpos — APPROVED** [review]

**Review by NTElissa — APPROVED** [review]

**Review by Muhet — APPROVED** [review]

**Review by kagaba-etienne — APPROVED** [review]
This is good Lievin!!

File: .env (hunk 2 above)
Comment (chanceDira) [reply .env]: can you use a variable name and a respective value as
e.g
NODE_ENV =development
PRO_DATABASE="postgress_url"
DEV_DATABASE="postgress_url"
TEST_DATABASE="postgress_url"

**Comment by nili-alu** [comment]
no conflict now

**Review by chanceDira — APPROVED** [review]
Good work @nili-alu @Vanessa-bitega

**Pull request merged**